lib/ally_steps.js

```diff
@@ -0,0 +1,18 @@
+const { accessibility } = require('taiko');
+const assert = require("assert");
+const MIN_ACCESSIBILITY_SCORE = process.env.min_accessibility_score || 80;
+
+step("Run a11y check", async () => {
```

Comment:
This will break existing test cases. Please add "Run a11y check" as an alias, to be backward compatible.